README.md

```diff
@@ -530,14 +530,28 @@ Uses [behave](https://github.com/behave/behave) to execute BDD scenarios that ar
 
 Builds the `sdist` and `wheel` archives of this project using `poetry build`. It also generates a `requirements.txt` file which is useful when installing the package in a Docker container where you may want to install the dependencies in a specific Docker layer to optimize caching.
 
+##### install #####
+Publishes the `pom.xml` for the module into your local Maven Repository ('~/.m2.repository`).
+
 ##### deploy #####
 
-Publishes the generated package archives to the specified PyPI repository (defined via **pypiRepoId** and **pypiRepoUrl**).  If the current Habushu module is a `SNAPSHOT` version, temporarily set the version of the package to the appropriate developmental version, publish it to the specified PyPI repository, and then reset the version to the original value.
+Publishes the generated package archives to the specified PyPI repository (defined via **pypiRepoId** and **pypiRepoUrl**).  
```

Comment:
A: add space after wheel files and put a title to better differentiate from deploy section.

Reply:
Fixed